data/navy watcher subplot

```diff
@@ -0,0 +1,673 @@
+	on fail
+		event "lab on watcher" 365
+
+mission "Knockout Gas 1
+	name "Top Secret Research"
+	landing
+	description "Bring some top secret research to the home of Lovelace Labs, <destination>."
+	cargo "research material" 15
+	to offer
+		has "Laboratory on Watcher 2: done"
+		"research teams" >= 4
+	source Watcher
+	destination Ada
+	cargo "opioid compoounds" 25
+	on offer
+		conversation
+			`Shortly after landing, you receive a message from Dr. Gonzales, from the research lab. "Captain <last>, what fantastic timing! Do you have time to drop by the lab to discuss a possible delivery job?"
+			choice
+				`	(Accept)`
+				`	(Decline)`
+					event "knockout gas available" 75
+					decline
+			`	As you take the short stroll from your ship to the research laboratory, you marvel at just how quickly Watcher has gone from uninhabitted moon to top-notch research facility. The amount of money that must be pouring into the research facilities boggles your mind, and you momentarily wonder which of the corporations hold the most strings. Certainly Lovelace has their hand in the ring - their ties to the University of Hermes are well-known. Rumors throughout the Dirt Belt hold that the Navy has their thumb in the pie as well. But many people seem convinced that Megaparsec, the Syndicate, Lionheart, and even the governments of the Deep are all engaged in a little "friendly" espionage out here on Watcher.`
+			branch both
+				has "Poachers 3: done"
+			`	You try to push the politics to the side of your mind as you are greeted at the entrance to the laboratory by Dr. Gonzales. "Good to see you again Captain <last>. As she leads you back to her office, she begins filling you in on more of the details: "We've been working with Lovelace Laboratories and have managed to isolate some of the opioid compounds in the Custodis cyathea rupertis. More intriguing, however, is that we believe we have found a way to weaponize these compounds, giving us a gas that could safely knock out a man in mere seconds."`
+			`	"Unfortunately," she continues, "Lovelace Labs isn't the only group interested in such a compound. We have already gotten some veiled threats from a few... interested parties. Because of the trouble with the Free Worlds, the Navy is unwilling to spare any ships for a delivery run, even though Lovelace expects the Navy to be their primary client for the final product. So it won't be easy," she concludes, "but if you can get it to Lovelace, they've agreed to give you 'first dibs' on the eventual product in addition to 90,000 credits."`
+				goto either
+			label both
+			`	You try to push the politics to the side of your mind as you are greeted at the entrance to the laboratory by Doctors Gonzales and Moore.`
+			`	Frank rushes forward to greet you. "<first>, it's so good to see you again! Can you believe they named this place after me?"`
+			`	Smiling, Christina retorts, "The nepotism is strong with this one."`
```

Comment:
Capitalise Nepotism then, maybe, otherwise grammar nazis. are going to jump on you